src/ui/public/field_editor/field_editor.html

```diff
@@ -94,7 +110,48 @@ <h4 class="hintbox-heading">
     </div>
 
     <div class="form-group">
-      <div ng-bind-html="editor.scriptingInfo" class="hintbox"></div>
+      <div class="hintbox">
+        <h4>
+          <i class="fa fa-question-circle text-info"></i> Scripting Help
+        </h4>
+
+        <p>
+          By default, Kibana scripted fields use <a target="_window" ng-href="{{editor.docLinks.painless}}">Painless <i class="fa-link fa"></i></a>, a simple and secure scripting language designed specifically for use with Elasticsearch.  To access values in the document use the following format:
```

Comment:
I'm getting a 404 errors for all the links in this section: Painless doc: https://www.elastic.co/guide/en/elasticsearch/reference/5.0/modules-scripting-painless.html 

Native Java APIs: https://www.elastic.co/guide/en/elasticsearch/reference/5.0/modules-scripting-painless.html#painless-api

Painless syntax: https://www.elastic.co/guide/en/elasticsearch/reference/5.0/modules-scripting-painless-syntax.html

Lucene expressions: https://www.elastic.co/guide/en/elasticsearch/reference/5.0/modules-scripting-expression.html